### ISSUE TITLE
Can only use one instance of the Knockout View Model viewer at a time

### ISSUE THREAD

**Issue by thecapdan** [BladeRunnerJS/brjs#1135]
See below:

https://github.com/BladeRunnerJS/brjs/issues/1135
![image](https://cloud.githubusercontent.com/assets/6085260/5473955/780eb94e-8605-11e4-8fa2-530dcfcd4138.png)

The initial theory is that JsTree relies on globals, so the second time we try to construct a tree it fails.

**Comment by ioanalianabalas** [comment]
Since #1196 is a duplicate of this, I am reposting the additional comments here:

When adding multiple blades for test in the BladesetWorkbench, the associated Knockout View Model will only be displayed for the first one. For the second blade, the panel will show but it will be empty.

If I try finding the associated view model with the 'find' button in the second blade, it will display the correct information, but it will remove all the content of the other Knockout View Model panel.

Basically, as it stands, we can only show information in one Knockout View Model at a time.

This is dependent on #1238 so I will postpone it until that one is fixed.
